chimney/src/main/scala/io/scalaland/chimney/dsl/TransformedNamesComparison.scala

```diff
@@ -44,6 +44,16 @@ object TransformedNamesComparison {
     def namesMatch(fromName: String, toName: String): Boolean = fromName.equalsIgnoreCase(toName)
   }
 
+  case object GeneratedProtobufEnumEquality extends TransformedNamesComparison {
```

Comment:
I would either name it `CamelCaseUnderscoreCaseEquality` (to e.g. use this with Java enums as well, maybe it should be a default for it? Maybe it should be named `JavaEnumAware`?) or keep it in `chimneyProtobuf` module. I wouldn't put some integration-specific conventions in the core package.